src/extensions/default/bramble/lib/ConsoleManager.js

```diff
@@ -0,0 +1,29 @@
+define(function (require, exports, module) {
+    "use strict";
+
+    var BrambleEvents = brackets.getModule("bramble/BrambleEvents");
+    var EditorManager = brackets.getModule("editor/EditorManager");
+    var CommandManager = brackets.getModule("command/CommandManager");
+    var Commands = brackets.getModule("command/Commands");
+
+    var ConsoleManagerRemote = require("text!lib/ConsoleManagerRemote.js");
+    var IFrameBrowser = require("lib/iframe-browser");
+
+    var isConsoleEnabled = false;
+
+    function getRemoteScript() {
+        return "<script>\n" + ConsoleManagerRemote + "</script>\n";
```

Comment:
Apologies. 
I'm working right now with using the same way as many other files for requiring a file, but it returns undefined in the <script> injection of that iframe. Suggestions? 
```js
var ConsoleManagerRemote = require("lib/ConsoleManagerRemote");
```
![image](https://cloud.githubusercontent.com/assets/14265337/23635305/e548f054-029c-11e7-9629-b30c12c6af64.png)